tests/test_instance.py

```diff
@@ -1,27 +1,46 @@
 import unittest
-from pynio import Instance
+from pynio import Instance, Block, Service
 from unittest.mock import MagicMock, patch
 from .mock import mock_service
 
 
+class MockInstance(Instance):
+
+    def __init__(self, host='127.0.0.1', port=8181, creds=None):
+        self._get_blocks = MagicMock()
+        self._get_blocks.return_value = {}, {}
+        self._get_services = MagicMock()
+        self._get_services.return_value = {}
+        super().__init__(host, port, creds)
+
+
 class TestInstance(unittest.TestCase):
-    def setUp(self):
```

Comment:
I've seen problems with mocking in setUp because you need to tearDown. And if the tests have problems then it never calls tearDown and weird things can start happening.